config/test.ex

```diff
@@ -0,0 +1,9 @@
+import Config
+
+if workos_api_key =
+     System.get_env("WORKOS_API_KEY") and workos_client_id = System.get_env("WORKOS_CLIENT_ID") do
```

Comment:
```suggestion
   System.get_env("WORKOS_API_KEY") and workos_client_id = System.get_env("WORKOS_CLIENT_ID") do
```

Reply:
That line was particularly long, and this was also the final formation by `mix format`. 

I ended up creating separate variables rather than performing pattern matching inline. https://github.com/workos/workos-elixir/pull/37/commits/c337813bc3b8df3a4fc03024090afac017470039